docs/CONFIG.md

```diff
@@ -22,6 +22,8 @@ Configuration lives in [config.yml](../config.yml).
   - `initial_backoff_duration`: The initial backoff duration (in seconds). Defaults to 5.
   - `backoff_multiplier`: The backoff multiplier. Defaults to 2.
   - `log_level`: Elasticsearch log level. Defaults to `INFO`.
+  - `language_code`: The language code. Defaults to `en`.
```

Comment:
maybe we should say here explicitely what it's used for:

`The language code used when creating a mapping for an index`
etc